lib/contents.php

```diff
@@ -99,14 +99,7 @@ function getContents(
         case 201:
         case 202:
             if (isset($result['headers']['cache-control'])) {
-                $cachecontrol = $result['headers']['cache-control'];
-                $lastValue = array_pop($cachecontrol);
-                $directives = explode(',', $lastValue);
-                $directives = array_map('trim', $directives);
-                if (in_array('no-cache', $directives) || in_array('no-store', $directives)) {
-                    // Don't cache as instructed by the server
-                    break;
-                }
+                // ignore cache-control
```

Comment:
Both vk and instagram sends the `cache-control: no-store` header.

Reply:
I think we should ignore cache header from server and cache anyways. But there might be reasons for respecting it which im unfamiliar with. I think ignoring cache headers from servers produces lesser requests towards servers and lessens the chance for anti-bot response.

Reply:
Cache in getContents is used to make requests with "if-modified-since" header. And if there is response with 304, cache is used to get body. Just like in browsers.

When servers responds with no-store, it means that browsers should not save downloaded pages to cache at all.